### DISCUSSION TITLE
Obtain a bond from a UniTensor by its label

### DISCUSSION THREAD

**Discussion by chiamin** [Cytnx-dev/Cytnx#209] [General]
I know using UniTensor.bonds() one can get all the bonds, but one needs to know the order to know which bond is which. Is there a way to get a bond by its label?

**Comment by kaihsin** [comment]
Good point. This should be easily done with adding:

1. UniTensor.bond(idx :int]) # by index
2. UniTensor.bond(lbl :str]) # by string
